databricks/koalas/namespace.py

```diff
@@ -2599,6 +2600,45 @@ def broadcast(obj) -> DataFrame:
     return DataFrame(obj._internal.with_new_sdf(F.broadcast(obj._internal.spark_frame)))
 
 
+def read_orc(path, columns: Optional[List[str]] = None, **options) -> "DataFrame":
+    """
+    Load an ORC object from the file path, returning a DataFrame.
+
+    Parameters
+    ----------
+    path : str
+        The path string storing the ORC file to be read.
+    columns : list, default None
+        If not None, only these columns will be read from the file.
+    options : dict
+        All other options passed directly into Spark's data source.
+
+    Returns
+    -------
+    DataFrame
+
+    Examples
+    --------
+    >>> ks.read_orc('data.orc')  # doctest: +SKIP
```

Comment:
can we make the test running via using pandas's `to_orc`?

Reply:
Oh,, just check that pandas doesn't have `to_orc`.